programmers/문자열-다루기-기본.js

```diff
@@ -0,0 +1,8 @@
+function solution(s) {
+    const S_LENGHT = s.length;
+    let temp = Number(s);
+    if ((S_LENGHT === 4 || S_LENGHT === 6) && temp === parseInt(s)) {
+        return true;
+    }
+    else return false; 
+}
```

Comment:
잘 풀었으나 if문에서 return을 할 경우 else를 사용하지 않아도 그외의 경우에 return을 함
그리고 문단 별로 줄바꿈 해주기!
```suggestion
function solution(s) {
    const S_LENGHT = s.length;
    let temp = Number(s);
    
    if ((S_LENGHT === 4 || S_LENGHT === 6) && temp === parseInt(s)) return true;
    return false; 
}
```